node_modules/col-core/lib/db.js

```diff
@@ -457,12 +466,17 @@ var setUpModel = function(sequelize) {
   /**
    * The `category` table keeps track of which categories assets can be tagged with
    *
-   * @property  (String)      display_name            The name of the category
+   * @property {String}       title                   The name of the category
+   * @property {Number}       [canvas_assignment_id]  The id of the assignment to which this category is linked
    */
   var Category = module.exports.Category = sequelize.define('category', {
-    'display_name': {
+    'title': {
       'type': Sequelize.STRING,
       'allowNull': false
+    },
+    'canvas_assignment_id': {
```

Comment:
Categories are optionally associated to an assignment in the course. When a new assignment is created in the assignments tool, this should be added as a category in the asset library. These categories should be highlighted as "special" categories in the UI.